common/viewer.h

```diff
@@ -188,12 +192,13 @@ namespace rs2
 
         std::shared_ptr<updates_model> updates;
 
-        bool draw_max_usable_range = true;
+        std::unordered_set<int> _hided_options;
+        reflectivity ref;
```

Comment:
`_`

Reply:
I thing `_reflectivity` is more readable (otherwise it could also mean 'reference')

Reply:
👍